crates/astria-conductor/src/bin/astria-conductor.rs

```diff
@@ -35,25 +27,24 @@ use tracing::{
     instrument,
 };
 
+const EXIT_CONFIG_CODE: i32 = 78;
+
 #[tokio::main]
 async fn main() -> Result<()> {
-    run().await?;
+    if let Err(e) = run().await {
+        eprintln!("Exited with error: {e}");
```

Comment:
```suggestion
        eprintln!("Exited with error:\n{e:?}");
```
Please print the entire eyre error backtrace.